.github/workflows/unit-tests.yml

```diff
@@ -1,5 +1,5 @@
 name: unit-tests
-on: [push, pull_request]
+on: [push, workflow_dispatch]
```

Comment:
We really do want `pull_request`. IMHO we should drop "push", as it's mostly just a resource waste -- these runs appear on your own fork on every `git push`, and usually few people look at them.

Reply:
oops, sorry, that was thinko, fixed